corehq/apps/registry/helper.py

```diff
@@ -21,3 +21,8 @@ def registry(self):
     @property
     def visible_domains(self):
         return {self.current_domain} | self.registry.get_granted_domains(self.current_domain)
+
+    @property
+    def participating_domains(self):
+        self.registry.check_ownership(self.current_domain)
+        return self.registry.get_participating_domains()
```

Comment:
The idea here was that this should only be used by the owning domain. Knowing which domains are participating is fine any of the domains can see that in the UI but since this is used for data access only the owning domain should have access to all domain data (in the backend).